sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberIntegrationTestBase.java

```diff
@@ -99,4 +100,11 @@ private String redact(String content, Matcher matcher, String replacement) {
 
         return content;
     }
+
+    protected PhoneNumberClientBuilder addLoggingPolicy(PhoneNumberClientBuilder builder, String testName) {
+        if (getTestMode() == TestMode.LIVE) {
+            builder.addPolicy(new CommunicationLoggerPolicy(testName));
```

Comment:
Is there any harm in adding the logging policy for `RECORD` or `PLAYBACK` tests? The policy only just prints a header to the console output right? It will make your tests simpler without having too many branches.

Reply:
You're right, I have removed this logic